tuist/tuist
# Fix warnings with including `.stencil` files when running `tuist edit`. Fixes #5603

Fixes #5603 

### Short description 📝

Add `.stencil` files as additional files instead of source files to avoid warnings during `tuist edit`

### How to test the changes locally 🧐

Run Tests. 

Run `tuist edit` on a fixture with custom resource synthesizers or scaffold templates that include stencil files.  Build the manifest project and no warnings should occur related to the stencil files.  See #5603 for more details.

### Contributor checklist ✅


### Reviewer checklist ✅
